### PR TITLE
feat: Updated email templates with user information

### PR DESCRIPTION
**Pull request by pmagictech**
**Description**
This pull request enhances the existing email templates by including the username of the recipient. This update aims to improve the personalization of the emails, making the communication more engaging and user-friendly. By making the User Entity object accessible in the email templates, Shield users can now easily add user-specific details, tailoring the emails to better suit their needs.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
Thank you for sending this PR.

We don't accept PRs without test code that proves the change works properly.
Please add test code.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#unit-testing

File: src/Views/Email/email_2fa_email.php [hunk 1]
```diff
@@ -23,9 +23,10 @@
...
     </table>
     <b><?= lang('Auth.emailInfo') ?></b>
+    <p><?= lang('Auth.username') ?>: <?= esc($user->username) ?></p>
     <p><?= lang('Auth.emailIpAddress') ?> <?= esc($ipAddress) ?></p>
     <p><?= lang('Auth.emailDevice') ?> <?= esc($userAgent) ?></p>
     <p><?= lang('Auth.emailDate') ?> <?= esc($date) ?></p>
 </body>
 
-</html>
+</html>
```
Comment (datamweb) [inline src/Views/Email/email_2fa_email.php]: Please use `LF` NOT `CRLF`.
See more information [understanding-line-break-types](https://dev.to/pranjal-barnwal/understanding-line-break-types-cr-lf-lf-and-cr-3lg2).

File: src/Views/Email/magic_link_email.php [hunk 2]
```diff
@@ -28,9 +28,10 @@
...
     </table>
     <b><?= lang('Auth.emailInfo') ?></b>
+    <p><?= lang('Auth.username') ?>: <?= esc($user->username) ?></p>
     <p><?= lang('Auth.emailIpAddress') ?> <?= esc($ipAddress) ?></p>
     <p><?= lang('Auth.emailDevice') ?> <?= esc($userAgent) ?></p>
     <p><?= lang('Auth.emailDate') ?> <?= esc($date) ?></p>
 </body>
 
-</html>
+</html>
```
Comment (datamweb) [inline src/Views/Email/magic_link_email.php]: Please use `LF` NOT `CRLF`.
See more information [understanding-line-break-types](https://dev.to/pranjal-barnwal/understanding-line-break-types-cr-lf-lf-and-cr-3lg2).

**Review by datamweb — CHANGES_REQUESTED** [review]
Thanks for PR.
You can get help from [Email Test](https://github.com/codeigniter4/shield/blob/aa40f9c121d9619470c743afc99426c2d29477a5/tests/Controllers/ActionsTest.php#L232-L251) for write the test.

**Review by datamweb — COMMENTED** [review]
Can you update description  https://github.com/codeigniter4/shield/blob/develop/docs/customization/views.md#customize-content and explain that user data(User Entity object) in email tpls is now available?

**Review by datamweb — APPROVED** [review]
LGTM🫡!

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@pmagictech Thank you!

**Comment by pmagictech** [comment]
@kenjis You are welcome

**Pull request merged**